mono/metadata/sgen-mono.c

```diff
@@ -2486,6 +2486,16 @@ mono_gc_get_los_limit (void)
 	return SGEN_MAX_SMALL_OBJ_SIZE;
 }
 
+int64_t
```

Comment:
`gint64`

Reply:
> `gint64`

@lewurm Why? We're C99, aren't we?

Reply:
@lambdageek consistency with the other function signatures in `boehm-gc.c` and `null-gc.c`